mlir/test/Interfaces/TilingInterface/tile-pad-using-interface.mlir

```diff
@@ -137,4 +188,14 @@ func.func @static_pad_tensor_outer_tiling(%input_tensor: tensor<7x9xf32>,
     } {__internal_transform__ = "pad_inner_tiling"} : tensor<7x9xf32> to tensor<15x16xf32>
```

Comment:
We don't need these attributes after switching to transform dialects. The new tests match the op directly.